src/operator/tensor/elemwise_binary_scalar_op_basic.cc

```diff
@@ -47,6 +47,26 @@ MXNET_OPERATOR_REGISTER_BINARY(_backward_rdiv_scalar)
 MXNET_OPERATOR_REGISTER_BINARY_SCALAR(_equal_scalar)
 .set_attr<FCompute>("FCompute<cpu>", BinaryScalarCompute<cpu, mshadow_op::eq>)
 .add_alias("_EqualScalar");
+    
+MXNET_OPERATOR_REGISTER_BINARY_SCALAR(_not_equal_scalar)
```

Comment:
OK, I will create separate files for the comparison operators and move the existing "equal" operator and the new comparison operators from the existing files to the new file.

- elemwise_binary_scalar_op_logic.cc
- elemwise_binary_broadcast_op_logic.cc
- elemwise_binary_scalar_op_logic.cu
- elemwise_binary_broadcast_op_logic.cu